core/lib/analytics/bigcommerce/index.ts

```diff
@@ -0,0 +1,61 @@
+import { cookies } from 'next/headers';
+import { validate as isUuid, v4 as uuidv4 } from 'uuid';
+
+const VISITOR_COOKIE_NAME = 'visitorId';
+const VISIT_COOKIE_NAME = 'visitId';
```

Comment:
Something I have been wanting to do is prefix cookies that Catalyst sets for better visibility what sets the cookie? Can we follow auth.js' footsteps here, e.g. `__Secure-authjs.session-token` and do something like `catalyst.visitId`.